hazelcast/src/main/java/com/hazelcast/util/QuickMath.java

```diff
@@ -204,4 +204,39 @@ public static String bytesToHex(byte[] in) {
         }
         return new String(hexChars);
     }
+
+    /**
+     * Utility method to compare integers
```

Comment:
You don't need to add 'utility method' because it provides no additional information. 

```
Compares 2 integers
```

What I'm missing is the explanation of the return values. So what do -1, 0 and 1 mean? Well.. 0 is easy.. but does 1 mean i1>i2 or i1<i2.

Perhaps call them first and second instead of those cryptic names?